ROLES.md

```diff
@@ -231,7 +231,7 @@ code, performing reviews, triaging issues, etc.
 
 * Sponsored by a working group lead with no objections from other leads
 
-If a maintainer becomes inactive in the project for an extended period of time, the individual will transition to being an
+If a maintainer becomes inactive in the project for over a year, the individual will transition to being an
```

Comment:
```suggestion
If a maintainer becomes inactive in the project for over a year, the individual will be asked to transition to an
```

Reply:
If we ask individuals, ii will cost a lot more efforts. And may not reach out to them

Reply:
I think the individual will be asked as part of a PR which aims to reach lazy consensus.  That's what we did this time and it worked.  We'll know if people are active or inactive and we can easily move someone back if they were inactive due to a holiday f.ex.